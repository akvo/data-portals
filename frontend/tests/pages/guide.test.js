import React from 'react'
import { render } from '@testing-library/react'
import Guide from '../../pages/guide'

describe('Gude page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Guide />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
