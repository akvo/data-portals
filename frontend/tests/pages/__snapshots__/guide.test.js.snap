// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gude page matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    Guide
  </h1>
  <strong>
    How does the portal work?
  </strong>
  <p>
    There are four WASH themes with corresponding subpages: Water, Water quality, Sanitation and Hygiene. On each subpage, there is a summary of widely used indicators and accompanying data visualisations. Next to each data visualisation, there is a question mark. Upon clicking the symbol, a text box will show with meta data and additional information on the methodology.
  </p>
  <p>
    Below we answered a number of 
    <strong>
      Frequently Asked Questions
    </strong>
     to ease the use of this portal. 
  </p>
  <h2>
    Frequently Asked Questions
  </h2>
  <div
    class="ant-collapse ant-collapse-icon-position-left"
  >
    <div
      class="ant-collapse-item"
    >
      <div
        aria-expanded="false"
        class="ant-collapse-header"
        role="button"
        tabindex="0"
      >
        <span
          aria-label="right"
          class="anticon anticon-right ant-collapse-arrow"
          role="img"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="right"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
            />
          </svg>
        </span>
        How can I contribute to the portal?
      </div>
    </div>
    <div
      class="ant-collapse-item"
    >
      <div
        aria-expanded="false"
        class="ant-collapse-header"
        role="button"
        tabindex="0"
      >
        <span
          aria-label="right"
          class="anticon anticon-right ant-collapse-arrow"
          role="img"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="right"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
            />
          </svg>
        </span>
        Is the data portal GDPDR compliant?
      </div>
    </div>
    <div
      class="ant-collapse-item"
    >
      <div
        aria-expanded="false"
        class="ant-collapse-header"
        role="button"
        tabindex="0"
      >
        <span
          aria-label="right"
          class="anticon anticon-right ant-collapse-arrow"
          role="img"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="right"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
            />
          </svg>
        </span>
        How can I find data with geo locations?
      </div>
    </div>
    <div
      class="ant-collapse-item"
    >
      <div
        aria-expanded="false"
        class="ant-collapse-header"
        role="button"
        tabindex="0"
      >
        <span
          aria-label="right"
          class="anticon anticon-right ant-collapse-arrow"
          role="img"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="right"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
            />
          </svg>
        </span>
        Can I download and use the data visualisations for own reports?
      </div>
    </div>
    <div
      class="ant-collapse-item"
    >
      <div
        aria-expanded="false"
        class="ant-collapse-header"
        role="button"
        tabindex="0"
      >
        <span
          aria-label="right"
          class="anticon anticon-right ant-collapse-arrow"
          role="img"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="right"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
            />
          </svg>
        </span>
        How often are data updated?
      </div>
    </div>
  </div>
</DocumentFragment>
`;
